shared-module/displayio/Display.c

```diff
@@ -281,12 +290,18 @@ STATIC bool _refresh_area(displayio_display_obj_t* self, const displayio_area_t*
             .x2 = clipped.x2,
             .y2 = clipped.y1 + rows_per_buffer * (j + 1)
         };
-        if (remaining_rows < rows_per_buffer) {
+        if (self->SH1107_addressing) {
+            // one row only for SH1107 in vertical (column) mode
+            subrectangle.y1 = clipped.y1 + j;
+            subrectangle.y2 = clipped.y1 + (j + 1);
```

Comment:
Is this needed still? rows_per_buffer is 1 so it should be equivalent to the starting value.

Reply:
Nope. Not required.  Tested on Grand Central M4 Express (slower than STM32F405... need to get that I2C solid!). Another commit coming...